### PR TITLE
Colab Links

### PR DESCRIPTION
**Pull request by sanket-kamthe**
Adds links to the notebooks so they can be directly opened in Colab

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,6 @@ The notebooks can be run live on [![Binder](https://mybinder.org/badge_logo.svg)
...
+|  Principal Component Analysis (PCA) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mml-book/mml-book.github.io/blob/master/tutorials/tutorial_pca.ipynb) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mml-book/mml-book.github.io/blob/master/tutorials/tutorial_pca.solution.ipynb) |
+| Gaussian Mixture Model (GMM) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mml-book/mml-book.github.io/blob/master/tutorials/tutorial_gmm.ipynb) | [ ![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mml-book/mml-book.github.io/blob/master/tutorials/tutorial_gmm.solution.ipynb )|
```
Comment (mpd37) [inline README.md:25]: remove space before closing bracket
(https://colab.research.google.com/github/mml-book/mml-book.github.io/blob/master/tutorials/tutorial_gmm.solution.ipynb )

File: README.md [hunk 2]
```diff
@@ -18,6 +18,6 @@ The notebooks can be run live on [![Binder](https://mybinder.org/badge_logo.svg)
...
-| Gaussian Mixture Model (GMM) | [ ![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg) ] ( https://colab.research.google.com/github/mml-book/mml-book.github.io/blob/master/tutorials/tutorial_gmm.ipynb | [ ![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg) ] ( https://colab.research.google.com/github/mml-book/mml-book.github.io/blob/master/tutorials/tutorial_gmm.solution.ipynb |
\ No newline at end of file
+| Linear regression  | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mml-book/mml-book.github.io/blob/master/tutorials/tutorial_linear_regression.ipynb) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/mml-book/mml-book.github.io/blob/master/tutorials/tutorial_linear_regression.solution.ipynb) |
```
Comment (mpd37) [inline README.md:23]: Linear regression -> Linear Regression

Comment (sanket-kamthe) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (sanket-kamthe) [reply README.md]: done

**Pull request merged**